admin-manual/maintenance/maintenance.rst

```diff
@@ -84,20 +86,64 @@ path of the AIP storage location you confirmed above.
 
 The command accepts the following parameters:
 
-* `[storage_location_path]` **[REQUIRED]**: Path where the AIP storage location
+* ``[storage_location_path]`` **[REQUIRED]**: Path where the AIP storage location
   is located in the local filesystem.
-* `--delete-all`: Removes the entire indexes to regenerate the mapping and
+* ``--delete-all``: Removes the entire indexes to regenerate the mapping and
   settings.
-* `--delete`: Removes matching AIP documents to avoid duplicates but keeps the
+* ``--delete``: Removes matching AIP documents to avoid duplicates but keeps the
   index mappings and settings.
-* `--uuid` [aip_uuid]: Index a single AIP from the storage location.
+* ``--uuid`` [aip_uuid]: Index a single AIP from the storage location.
 
 It can be executed multiple times with different paths to index multiple AIP
 storage locations.
 
+.. _aip-indexes-api:
+
+**Rebuild AIP indexes using Storage Service API**
+
+This command uses the Storage Service API to determine which stored AIPs and
+AICs need to be reindexed (based on status and origin pipeline), and then
+reindexes those AIPs and AICs from temporarily downloaded copies of the their
```

Comment:
I think "the" should be removed from "the their" at the end of this line

Reply:
Nice catch thanks. Change made. Merging...